src/dom-utils.ts

```diff
@@ -125,9 +155,18 @@ export function isFocusable(el: HTMLElement) {
  * ```
  */
 function canHaveFocusableChildren(el: HTMLElement) {
-  // Elemments matching this selector are either hidden entirely from the user,
-  // or are visible but unavailable for interaction. Their descentants can never
-  // receive focus.
+  /**
```

Comment:
We’ve been a little inconsistent with comment styles. I think we use block-style comments *before* functions but inline-style comments *inside* functions?